controllers/constant/constants.go

```diff
@@ -17,12 +17,16 @@
 package constant
 
 const (
-	NamespaceScopeManagedPrefix  = "nss-managed-role-from-"
-	NamespaceScopeConfigmapName  = "namespace-scope"
-	NamespaceScopeFinalizer      = "finalizer.nss.operator.ibm.com"
-	NamespaceScopeLabel          = "managedby-namespace-scope"
-	DefaultRestartLabelsKey      = "intent"
-	DefaultRestartLabelsValue    = "projected"
-	NamespaceScopeServiceAccount = "ibm-namespace-scope-operator"
-	InjectorMark                 = "nss.operator.ibm.com/managed-operators"
+	NamespaceScopeManagedPrefix           = "nss-managed-role-from-"
+	NamespaceScopeConfigmapName           = "namespace-scope"
+	NamespaceScopeFinalizer               = "finalizer.nss.operator.ibm.com"
+	NamespaceScopeLabel                   = "managedby-namespace-scope"
+	DefaultRestartLabelsKey               = "intent"
+	DefaultRestartLabelsValue             = "projected"
+	NamespaceScopeServiceAccount          = "ibm-namespace-scope-operator"
+	InjectorMark                          = "nss.operator.ibm.com/managed-operators"
+	NamespaceScopeConfigmapLabelKey       = "namespace-scope-configmap"
+	NamespaceScopeConfigmapNSSLabelValue  = "ibm-common-services-namespace-scope"
+	NamespaceScopeConfigmapODLMLabelValue = "ibm-common-services-odlm-scope"
```

Comment:
@ZhuoxiLi We should not hard code the label here.
We should combine the NSS instance.Namespace and instance.Spec.ConfigmapName to build to key.

In the function `GetRolesFromNamespace`, We should list all the NSS instances in the namespace and go through these instances to find out the key of the label.